### PR TITLE
Removing "experimental" messaging from LotL docs

### PR DESCRIPTION
**Pull request by ajosh0504**
## Related to
[security-team/8022](https://github.com/elastic/security-team/issues/8022)

## What I did in this PR
This PR removes any "experimental" messaging for the `problemchild` integration package.

### CODE REVIEW

**Review by sodhikirti07 — APPROVED** [review]
LGTM!

File: packages/dga/changelog.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@
   changes:
     - description: Removing detection rules from the package, bumped license and format versions, subscription tier
       type: enhancement
-      link: https://github.com/elastic/integrations/pull/7902
+      link: https://github.com/elastic/integrations/pull/8473
```
Comment (susan-shu-c) [inline packages/dga/changelog.yml:6]: Did you mean to copy this chunk and then add `version: "2.1.1"`? I only see `version: 
2.0.0
` in this file. Also, the manifest says `2.1.1` so I think you're missing that in this changelog

**Review by susan-shu-c — CHANGES_REQUESTED** [review]

File: packages/dga/changelog.yml (hunk 1 above)
Comment (susan-shu-c) [inline packages/dga/changelog.yml:6]: Is this change in `dga` changelog necessary?

File: packages/problemchild/changelog.yml [hunk 2]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.1.1"
+  changes:
+    - description: Remove "experimental" messaging from docs
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/8106
```
Comment (susan-shu-c) [inline packages/problemchild/changelog.yml:6]: I think this isn't the correct PR?

**Review by susan-shu-c — CHANGES_REQUESTED** [review]

**Review by susan-shu-c — APPROVED** [review]

**Pull request merged**